REQUIREMENTS.md

```diff
@@ -76,7 +76,7 @@ generated requirements.
 
 Your second command is actually more like:
 ```
-$ pip-compile -o requirements.txt requirements.txt requirements.in
+$ pip-compile -o requirements.txt requirements.in
```

Comment:
If I understand this correctly this line is trying to say that:

`pip-compile -o requirements.txt requirements.in`

is equivalent to:

`pip-compile -o requirements.txt requirements.txt requirements.in`

And is _not_ saying that you should run it like this to update the requirements.txt.